docs/platform-integration/sensors/geocoding.md

```diff
@@ -0,0 +1,58 @@
+---
+title: "Geocoding"
+description: "Learn how to use the .NET MAUI Geocoding class in the Microsoft.Maui.Devices.Sensors namespace. This class provides APIs to both geocode a placemark to a positional coordinate, and reverse geocode coordinates to a placemark."
+ms.date: 05/12/2022
+no-loc: ["Microsoft.Maui", "Microsoft.Maui.Devices", "Microsoft.Maui.Devices.Sensors"]
+---
+
+# Geocoding
+
+This article describes how you can use the .NET Multi-platform App UI (.NET MAUI) `IGeocoding` interface. This interfaces provides APIs to geocode a placemark to a positional coordinates and reverse geocode coordinates to a placemark. The `IGeocoding` interface is exposed through the `Geocoding.Default` property.
+
+[!INCLUDE [docs under construction](~/includes/preview-note.md)]
+
+The `Geocoding` and `IGeocoding` types are available in the `Microsoft.Maui.Devices.Sensors` namespace.
+
+## Get started
+
+To access the **Geocoding** functionality the following platform-specific setup is required.
+
+<!-- markdownlint-disable MD025 -->
+# [Android](#tab/android)
+
+No setup is required.
+
+# [iOS](#tab/ios)
+
+No setup is required.
+
+# [Windows](#tab/windows)
+
+A Bing Maps API key is required to use geocoding functionality. Sign up for a free [Bing Maps](https://www.bingmapsportal.com/) account. Under **My account** > **My keys**, create a new key and fill out information based on your application type, which should be **Public Windows App (UWP, 8.x, and earlier)**.
```

Comment:
Should this now be `UseMapServiceToken` in `ConfigureEssentials`?

Reply:
@davidbritch I'm not sure what you're talking about. I did go check the bing maps section and the app type is now **Windows Application** which I just updated in the article.

If you mean the code blow this, the windows specific code: `Platform.MapServiceToken = "YOUR-KEY-HERE";` I don't know. If I switch my platform to Windows, intellisense and the object browser won't change off of android. So I can't compile or tell what it should be; I can't test it either.

Reply:
Looks like this doc still needs an update. I'll resolve this issue, but add a new comment on the specific part of the doc.